server/keyshare/keyshareserver/server.go

```diff
@@ -344,8 +344,16 @@ func (s *Server) handleVerify(w http.ResponseWriter, r *http.Request) {
 
 	var username string
 	if msg.AuthResponseJWT == "" {
+		if r.URL.Path != "/users/verify/pin" {
```

Comment:
would be nice to extract these strings to a variable since they're being used twice in this file and if the endpoint name changes in the router, it should change here, too.